js/utils/helpers.js

```diff
@@ -1969,11 +1973,20 @@
                                 captionHTML = '<a href="' + enlargeLink + '" target="_blank">'  + captionHTML + '</a>';
                             }
 
+                            // Checks for a width being defined. If it's defined and not a number, assume it has `px` or `%` appended already and use as is.
+                            // If width is defined and is a number, assume it's in pixels and append `%`.
+                            // If no width, use "100%"
+                            var contentsWidth = 'style="width: ' + (frameWidth ? (frameWidth + (isNaN(parseInt(frameWidth)) ? "" : "px")) : "100%")+ ';"';
+
+                            // text align to pull button to the right
+                            var buttonHtml = '<div class="iframe-btn-container pull-right"><a class="chaise-btn chaise-btn-secondary chaise-btn-iframe" href="' + iframeSrc + '" target="_blank"><span class="glyphicon glyphicon-fullscreen"></span> Full screen</a></div>';
```

Comment:
Since `pull-right` is a class specific too Bootstrap (and therefore Chaise environment) let's just remove it and make sure the `iframe-btn-container` rule in Chaise does the `float: right`